tidy3d/components/data/monitor_data.py

```diff
@@ -1356,7 +1397,23 @@ class PermittivityData(PermittivityDataset, AbstractFieldData):
 
 
 class ModeData(MonitorData):
-    """Data associated with a :class:`.ModeMonitor`: modal amplitudes and propagation indices.
+    """
+    Data associated with a :class:`.ModeMonitor`: modal amplitudes and propagation indices.
+
+    Notes
+    -----
+
+        The data is stored as a `DataArray <https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html>`_
+        object using the `xarray <https://docs.xarray.dev/en/stable/index.html>`_ package.
+
+        The mode monitor data contains the complex effective indices and the complex mode amplitudes at the monitor
+        position calculated by mode decomposition. The data structure of the complex effective
+        indices :attr`n_complex` contains two coordinates: ``f`` and ``mode_index``, both of which are specified when
+        defining the :class:``ModeMonitor`` in the simulation.
```

Comment:
single backslashes in these class links. (and the paragraph below, maybe grep for this everywhere)